sdk/tables/azure-data-tables/CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 ## 12.0.0-beta.2 (Unreleased)
 
-### Fixes
+### Changed
+
+- The `getEntity` methods have gained the `select` query option to allow for more efficient existence checks for a table entity #15289
```

Comment:
I'd add a link to the issue rather than the number.

Reply:
Fixed